pkg/interface/src/views/apps/chat/components/lib/ChatWindow.tsx

```diff
@@ -127,7 +127,8 @@ export default class ChatWindow extends Component<ChatWindowProps, ChatWindowSta
     const { isChatMissing, history, envelopes, mailboxSize, stationPendingMessages, unreadCount } = this.props;
 
     if (isChatMissing) {
-      history.push("/~chat");
+      //  TODO: fix this and push a different route
+      history.push("/~404");
```

Comment:
TODO

Reply:
Actually, I think this is the correct behavior here, to show a 404.

Reply:
Agreed